module/VuFind/src/VuFind/ILS/Driver/AbstractBase.php

```diff
@@ -66,6 +69,117 @@ public function setConfig($config)
         $this->config = $config;
     }
 
+    /**
+     * Create a patron array according to patronLogin specs defined in the documentation.
+     * Each value is trimmed if they are type of string.
```

Comment:
Minor clarification:

```suggestion
     * Each value is trimmed if they are string-typed.
```